src/assets/scripts/client/aircraft/FlightManagementSystem/RouteModel.js

```diff
@@ -1020,10 +1020,18 @@ export default class RouteModel extends BaseModel {
         }
 
         if (convergentLegModel.isSidLeg) {
+            if (_first(convergentLegModel.waypoints).name === endWaypointName) {
```

Comment:
you should pull this out to a `const` so that we can reuse it:
```js
const firstWaypointName = _first(convergentLegModel.waypoints).name;
```

Reply:
done